assets/js/timetree.js

```diff
@@ -213,11 +233,11 @@ function TreeGraph({nodes, links, centuries}) {
     // .alphaDecay(0.2)
     .force("collide", d3.forceCollide().radius(18))
     // NOTE: may want to adjust to make variable by node type
-    .force("link", d3.forceLink(links))
+    .force("link", d3.forceLink(links).strength(link => 0.8))
```

Comment:
Does this need to be parameterized/set elsewhere?

Reply:
good thought, maybe — right now I'm just playing with the forces a bit to see how much I can use them to tweak the layout & placement.  I'm currently thinking I'll need to adjust the century pseudo-nodes anyway (i.e., add more nodes for branch+century if there are more nodes), which may negate the need for this adjustment; the default link strength is based on degree, which isn't great for our current leaf+node structure.